pilot/platform/kube/inject/testdata/frontend.yaml.injected

```diff
@@ -117,7 +117,6 @@ spec:
       volumes:
       - emptyDir:
           medium: Memory
-          sizeLimit: "0"
```

Comment:
SizeLimit was changed to a pointer value (see https://github.com/kubernetes/kubernetes/pull/50163)